control-flow/includes/2-switch.md

```diff
@@ -0,0 +1,209 @@
+# Use Switch Statements
+
+Like other programming languages, Go also has support for `switch` statements. When you need to have several `if` statements chained, a better approach is to use `switch` statements. Not only you'll avoid having too many `if` statements that could make your code harder to read and maintain, but you can also have complicated conditions. We'll give them a look in the below sections.
+
+# Basic Switch Syntax
+
+Like the `if` statement, you don't need parenthesis for the `switch` condition. At its simplest form, a `switch` statement looks like this:
+
+```go
+package main
+
+import (
+    "fmt"
+    "math/rand"
+    "time"
+)
+
+func main() {
+    sec := time.Now().Unix()
+    rand.Seed(sec)
+    i := rand.Int31n(10)
+
+    switch i {
+    case 0:
+        fmt.Print("zero...")
+    case 1:
+        fmt.Print("one...")
+    case 2:
+        fmt.Print("two...")
+    }
+
+    fmt.Println("ok")
+}
+```
+
+Run several times above code, and you'll see a different output every time (If you run it in the Go Playground, you'll get the same result every time, and that's one of the service's limitations).
+
+Go executes each case of the `switch` statement until it finds a match for the condition. However, notice that the previous code doesn't cover all possible cases of the `num` variable's values. For instance, if `num` ends up being `5`, the program's output will simply be `ok`. Alternatively, you can be more specific about the default use case and include it like this:
+
+```go
+switch i {
+case 0:
+    fmt.Print("zero...")
+case 1:
+    fmt.Print("one...")
+case 2:
+    fmt.Print("two...")
+default:
+    fmt.Print("no match...")
+}
+```
+
+Notice that for the `default` case, you don't write an expression to validate.
```

Comment:
Got it, I've included a change to reflect that

> Notice that for the `default` case, you don't write an expression to validate. **Instead, you simply include the `i` variable because you'll validate its value in the different `case` statements**.